frontend/src/app/App.scss

```diff
@@ -41,6 +41,12 @@ body,
       inset-inline-start: 12px;
     }
   }
+  .pf-topology-container {
+    overflow-y: hidden;
+    .pf-v5-c-drawer__panel.pf-m-resizable {
+      min-width: 350px;
+    }
+  }
```

Comment:
Is this needed? doesnt the drawer already have a min width

Reply:
Removed this styling